### PR TITLE
Improve French I18N

### PR DESCRIPTION
**Pull request by jeffmaury**
Ready for review

### CODE REVIEW

**Comment by feloy** [comment]
> Draft as I want to make sure I'm on right path

Yes, that seems good to me up to now.
Thanks for your help on this translation

**Comment by maciaszczykm** [comment]
@feloy can you review it?

**Comment by feloy** [comment]
> @feloy can you review it?

Yes, I will do

File: modules/web/i18n/fr/messages.fr.xlf [hunk 1]
```diff
@@ -1726,23 +1722,23 @@
       </trans-unit>
       <trans-unit id="6852503341223458730" datatype="html">
         <source>Persistent Volumes</source>
-        <target state="new">Persistent Volumes</target>
+        <target>Volumes persistents</target>
```
Comment (feloy) [inline modules/web/i18n/fr/messages.fr.xlf:1725]:
```suggestion
        <target>Volumes persistants</target>
```

File: modules/web/i18n/fr/messages.fr.xlf [hunk 2]
```diff
@@ -2206,31 +2202,31 @@
...
-        <target state="new">Download logs file</target>
+        <target>Télécharger les fichiers journaux</target>
       </trans-unit>
       <trans-unit id="8444287437490086299" datatype="html">
         <source>Endpoints <x id="START_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&quot;\uFFFD#2\uFFFD&quot;"/>open_in_new<x id="CLOSE_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&quot;\uFFFD/#2\uFFFD&quot;"/></source>
-        <target state="new">Endpoints <x id="START_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&lt;mat-icon class=&quot;external-link-icon kd-clickable&quot; i18n-matTooltip matTooltip=&quot;Endpoint links are external links that will be open in a new tab.&quot;&gt;"/>open_in_new<x id="CLOSE_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&lt;/mat-icon&gt;"/></target>
+        <target>Points de terminaison <x id="START_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&lt;mat-icon class=&quot;external-link-icon kd-clickable&quot; i18n-matTooltip matTooltip=&quot;Les liens de point de terminaison sont des liens externes qui seront ouvert dans un nouvel onglet.&quot;&gt;"/>open_in_new<x id="CLOSE_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&lt;/mat-icon&gt;"/></target>
```
Comment (feloy) [inline modules/web/i18n/fr/messages.fr.xlf:2221]:
```suggestion
        <target>Points de terminaison <x id="START_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&lt;mat-icon class=&quot;external-link-icon kd-clickable&quot; i18n-matTooltip matTooltip=&quot;Les liens de point de terminaison sont des liens externes qui seront ouverts dans un nouvel onglet.&quot;&gt;"/>open_in_new<x id="CLOSE_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&lt;/mat-icon&gt;"/></target>
```

File: modules/web/i18n/fr/messages.fr.xlf [hunk 3]
```diff
@@ -2282,59 +2278,59 @@
...
       </trans-unit>
       <trans-unit id="8802433277842246383" datatype="html">
         <source>Started At</source>
-        <target state="new">Started At</target>
+        <target>Commencé à</target>
       </trans-unit>
       <trans-unit id="8825668517883103754" datatype="html">
         <source>Cluster Roles</source>
-        <target>Cluster Roles</target>
+        <target>Rôles du cluster</target>
```
Comment (feloy) [inline modules/web/i18n/fr/messages.fr.xlf:2333]: ?
```suggestion
        <target>Rôles de cluster</target>
```

**Review by feloy — COMMENTED** [review]
Some fixes, and propositions
Thanks for your help on this translation

**Pull request merged**